lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
@exogen is correct about mutating `arguments` if already set. I agree that "we don't really care" since we're now within a function we own.

Next question -- on second thought, if someone passes `null|undefined` context should we just pass that through without the `context || this`?

/cc @baer @exogen